data/avro-schema/src/main/resources/avro/net/corda/data/flow/state/db/Query.avsc

```diff
@@ -0,0 +1,39 @@
+{
+  "type": "record",
+  "name": "Query",
+  "namespace": "net.corda.data.flow.state.db",
```

Comment:
Should we make this `PersistenceState` or `DatabaseState`? To keep it consistent with `SessionState`.

Reply:
After standup discussion i'll rename query to persistenceState